src/bin-utils/index.test.js

```diff
@@ -305,6 +311,69 @@ test(
   },
 )
 
+test('specificHelpScript: returns no script available', () => {
+  const config = {scripts: {}}
+  const message = specificHelpScript(config, 'foo')
+  const expected = chalk.yellow('Script foo does not exist')
+  expect(message).toBe(expected)
+})
+
+test(
+  'specificHelpScript: do not display script with flag hiddenFromHelp=true',
+  () => {
+    const config = {
+      scripts: {
+        foo: {
+          description: 'the foo script',
+          script: 'echo "foo"',
+          hiddenFromHelp: true,
+        },
+      },
+    }
+    const message = specificHelpScript(config, 'foo')
+    const expected = chalk.yellow('Script foo does not exist')
+    expect(message).toBe(expected)
+  },
+)
+
+test('specificHelpScript: formats a nice message', () => {
+  const config = {
```

Comment:
Hmmm... It would be easier to review if you made a separate PR to refactor things, then update this PR to use that.

Thank you so much for your work on this. It's incredibly valuable and helpful. This project needs a bit of love and you're giving it that. So thank you very much.